src/NuGet.Core/NuGet.Packaging/compiler/resources/nuspec.xsd

```diff
@@ -153,6 +153,7 @@
                               </xs:sequence>
                             </xs:complexType>
                           </xs:element>
+                          <xs:element name="icon" maxOccurs="1" minOccurs="0" type="xs:string" />
```

Comment:
Actually I think you should move this up next to the other simple types. 
So somewhere between serviceable and repository.

Reply:
I think it's a good idea.